src/middlewared/middlewared/service.py

```diff
@@ -871,6 +873,90 @@ def ping(self):
         """
         return 'pong'
 
+    def _ping_host(self, host, timeout, counter=3):
+        if osc.IS_LINUX:
+            process = run(['ping', '-4', '-w', f'{timeout}', '-c', f'{counter}', host])
```

Comment:
Specifying `count` on Linux is ignored when you specify `-w` so let's remove the `counter` keyword please.

To make it even more confusing, freeBSD has differing behavior when specifying a timeout and a count.

On freeBSD, it will send `count` packets and wait `timeout` which further validates the reason why we should remove the `counter` keyword.